src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
Or maybe it should be like in the case of !FEATURE_STUBS_AS_IL. That code builds the MetaSig from pStubMD when it is not NULL, but this way if it is NULL:
MetaSig msig(GetSignature(), GetModule(), NULL);